opm/simulators/utils/readDeck.cpp

```diff
@@ -477,7 +477,7 @@ Opm::setupLogging(const int          mpi_rank_,
     return output;
 }
 
-void Opm::readDeck(Opm::Parallel::Communication    comm,
+void Opm::readDeck(Opm::Parallel::Communication&    comm,
```

Comment:
seems unrelated. these structs are very cheap to copy so that's been the preferred semantics.